analyzers/src/SonarAnalyzer.VisualBasic/Extensions/MethodBlockSyntaxExtensions.cs

```diff
@@ -22,6 +22,9 @@ namespace SonarAnalyzer.Extensions;
 
 public static class MethodBlockSyntaxExtensions
 {
+    public static bool IsDeclare(this MethodBlockSyntax methodBlock) =>
+        methodBlock.SubOrFunctionStatement.Modifiers.Any(SyntaxKind.DeclareKeyword);
```

Comment:
This looks wrong to me and isn't under test. I would expect something like this:
```suggestion
    public static bool IsDeclare(this DeclareStatementSyntax method) =>
        true;
```
or this:
```cs
    public static bool IsDeclare(this MethodBaseSyntax method) =>
        method is DeclareStatementSyntax;
```